cmd/notification.go

```diff
@@ -1101,19 +1103,78 @@ func (sys *NotificationSys) GetBandwidthReports(ctx context.Context, buckets ...
 	return consolidatedReport
 }
 
-// GetClusterMetrics - gets the cluster metrics from all nodes excluding self.
-func (sys *NotificationSys) GetClusterMetrics(ctx context.Context) <-chan Metric {
+// GetBucketMetrics - gets the cluster level bucket metrics from all nodes excluding self.
+func (sys *NotificationSys) GetBucketMetrics(ctx context.Context) <-chan Metric {
```

Comment:
```suggestion
func (sys *NotificationSys) GetPeerBucketMetrics(ctx context.Context) <-chan Metric {
```

as it only collects bucket metrics from peers?

Reply:
Yeah we but we don't call it that way. It's all Peers do it is called a higher level name.